salt/modules/zypper.py

```diff
@@ -1729,10 +1827,11 @@ def clean_locks():
     '''
     LCK = "removed"
     out = {LCK: 0}
-    if not os.path.exists("/etc/zypp/locks"):
+    locks = os.path.join(root, os.path.relpath(LOCKS, start='/')) if root else LOCKS
```

Comment:
Technically, `os.path.sep`, not `'/'`. :wink:

Reply:
indeed!